package.json

```diff
@@ -17,30 +17,35 @@
     "@girder/oauth-client": "^0.8.0",
     "@mdi/font": "^6.5.95",
     "@sentry/vue": "^6.16.1",
+    "@types/geojson": "^7946.0.8",
```

Comment:
Is it possible to move this to `devDependencies` instead, so it's not included in the minified production build?

Reply:
resolved with 38eb4c6fb645448f9e4199772d46bfbca8cac6fa